Dockerfile

```diff
@@ -33,8 +33,10 @@ RUN [ -s /home/extractor/packages.txt ] && \
 
 RUN [ -s /home/extractor/requirements.txt ] && \
     (echo 'Install python modules' && \
-    python3 -m pip install -U --no-cache-dir pip && \
-    python3 -m pip install --no-cache-dir setuptools && \
+    python3 -m pip install --upgrade --no-cache-dir pip && \
+    python3 -m pip install --upgrade --no-cache-dir setuptools && \
+    python3 -m pip install --upgrade --no-cache-dir numpy && \
+    python3 -m pip install --no-cache-dir pygdal==2.2.2.* && \
```

Comment:
Can these be put into `requirements.txt`?

Reply:
This was made so that the pylint workflow would work separately from the dockerfile since gdal was not working with both (version mismatches). I went based off of https://cyverse.slack.com/archives/D0103DDM37E/p1594746216012900 . It was mentioned to install the needed packages/requirements in the .yaml file separately for now